### ISSUE TITLE
Error on upgrade -  IllegalArgumentException

### ISSUE THREAD

**Issue by EvilOlaf** [kangarko/ChatControl-Pro#110]
Tried updating from free 5.3.6 to pro 7.0.18

```
[...]
[14:17:55] [Server thread/INFO]:  [0;31;1m[0;37;1mStarting 7.0.0, '[0;33;1mMessages[0;37;1m' section[m
[14:17:55] [Server thread/INFO]:  [0;31;1m[0;37;1mfrom settings.yml has been removed[m
[14:17:55] [Server thread/INFO]:  [0;31;1m[0;37;1mand it is now found in messages.yml[m
[14:17:55] [Server thread/INFO]:  [0;31;1m [m
[14:17:55] [Server thread/INFO]:  [0;31;1m[0;33;1mSection here is no longer functional.[m
[14:17:55] [Server thread/INFO]: [0;37;22m*-----------------------------------------------*[m
[14:17:55] [Server thread/INFO]: [0;33;1mSaved updated file: settings.yml (# Comments removed)[m
[14:17:55] [Server thread/INFO]: [0;37;22m*-----------------------------------------------*[m
[14:17:55] [Server thread/INFO]:  [0;31;1m[0;31;1m@@@@@@ @@@@@@ [0;33;22mATTENTION [0;31;1m@@@@@@ @@@@@@[m
[14:17:55] [Server thread/INFO]:  [0;31;1mIf you upgraded from the free version[m
[14:17:55] [Server thread/INFO]:  [0;31;1myou need to update your variables. See:[m
[14:17:55] [Server thread/INFO]:  [0;31;1m[0;33;1mhttps://goo.gl/dYv1TE[m
[14:17:55] [Server thread/INFO]:  [0;31;1m[0;31;1m@@@@@@ @@@@@@ @@@@@@ @@@@@@ @@@@@@ @@@@@@[m
[14:17:55] [Server thread/INFO]: [0;37;22m*-----------------------------------------------*[m
[14:17:55] [Server thread/WARN]: java.lang.IllegalArgumentException: Missing variables/global.txt file!
[14:17:55] [Server thread/WARN]: 	at org.apache.commons.lang.Validate.isTrue(Validate.java:136)
[14:17:55] [Server thread/WARN]: 	at nul.final...\\....if.NUl.private(SourceFile:87)
[14:17:55] [Server thread/WARN]: 	at nul.final...\\....if.Com3.const(SourceFile:54)
[14:17:55] [Server thread/WARN]: 	at nul.final...\\....if.com6.onEnable(SourceFile:56)
[14:17:55] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321)
[14:17:55] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340)
[14:17:55] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405)
[14:17:55] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:357)
[14:17:55] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:317)
[14:17:55] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.MinecraftServer.s(MinecraftServer.java:414)
[14:17:55] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.MinecraftServer.k(MinecraftServer.java:378)
[14:17:55] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:333)
[14:17:55] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:263)
[14:17:55] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:525)
[14:17:55] [Server thread/WARN]: 	at java.lang.Thread.run(Unknown Source)
[14:17:55] [Server thread/INFO]: [0;31;1m  _   _                       _ [m
[14:17:55] [Server thread/INFO]: [0;31;1m  | | | | ___   ___  _ __  ___| |[m
[14:17:55] [Server thread/INFO]: [0;31;1m  | |_| |/ _ \ / _ \| '_ \/ __| |[m
[14:17:55] [Server thread/INFO]: [0;31;1m  |  _  | (_) | (_) | |_) \__ \_|[m
[14:17:55] [Server thread/INFO]: [0;31;22m  |_| |_|\___/ \___/| .__/|___(_)[m
[14:17:55] [Server thread/INFO]: [0;31;22m                    |_|          [m
[14:17:55] [Server thread/INFO]: [0;31;22m!-----------------------------------------------------![m
[14:17:55] [Server thread/INFO]:  [0;31;1mError loading ChatControl, plugin is disabled![m
[14:17:55] [Server thread/INFO]:  [0;31;1mRunning on 1.8.8-R0.1-SNAPSHOT (v1_8_R3) & Java 1.8.0_101[m
[14:17:55] [Server thread/INFO]: [0;31;22m!-----------------------------------------------------![m
[14:17:55] [Server thread/INFO]:  [0;31;1mError: Missing variables/global.txt file![m
[14:17:55] [Server thread/INFO]: [0;31;22m!-----------------------------------------------------![m
[14:17:55] [Server thread/INFO]: [ChatControl] Disabling ChatControl v7.0.18
[...]
```

**Comment by kangarko** [comment]
Hey, I am not at my workspace, in due time create manually the file variables/global.txt and restart the server.

**Comment by EvilOlaf** [comment]
Relax. I just tested this with a full copy of the server in an experimental environment.
When a fix has been released I will redo that again to confirm.

**Comment by kangarko** [comment]
Okay okay :) The file has been created in the 6th generation and renamed in the 7th generation. So the updater seems not to recognize that there was no file back in the 5th generation. I did not experience that problem during testing. Thanks for reporting, will get it it soon.

**Comment by kangarko** [comment]
Fixed in 7.0.19

**Comment by EvilOlaf** [comment]
Nice. Will give it another try later.
